bad string creation in legacy camera app

typhartez opened the pull request:
Fixes issue #61

lickx commented:
I think there was a codepage problem with the whitespace, I modified it a tiny bit so it compiles

typhartez commented:
Oh, for git tree editing, I usually use Atom. I thought it correctly
converted a whole file to one encoding (utf-8) when saving. I will check
that.

2018-02-05 15:16 GMT+01:00 lickx <notifications@github.com>:

> I think there was a codepage problem with the whitespace, I modified it a
> tiny bit so it compiles
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lickx/oscollar/pull/62#issuecomment-363096581>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/APytPkhDomeiBCqliA8PBXgnBJthOQeuks5tRw0qgaJpZM4R5Vpy>
> .
>



-- 
*It is easier to disintegrate an atom than a prejudice* (Albert Einstein)
